crates/unftp-auth-jsonfile/src/lib.rs

```diff
@@ -210,7 +212,40 @@ struct UserCreds {
 impl JsonFileAuthenticator {
     /// Initialize a new [`JsonFileAuthenticator`] from file.
     pub fn from_file<P: AsRef<Path>>(filename: P) -> Result<Self, Box<dyn std::error::Error>> {
-        let json: String = fs::read_to_string(filename)?;
+        let mut f = fs::File::open(&filename)?;
+
+        // The credentials file can be plaintext, gzipped, or gzipped+base64-encoded
```

Comment:
Perhaps this explanation is nice to add to the public docs e.g. https://unftp.rs/server/jsonconfig

Reply:
Good point! Yes